web/src/components/molecules/Request/Details/SidebarWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "@emotion/styled";
+
+import SidebarCard from "@reearth-cms/components/molecules/Request/Details/SidebarCard";
+import { Request } from "@reearth-cms/components/molecules/Request/types";
+import { useT } from "@reearth-cms/i18n";
+
+export type Props = {
+  currentRequest?: Request;
+};
+
+const RequestSidebarWrapper: React.FC<Props> = ({ currentRequest }) => {
+  const t = useT();
+
+  return (
+    <SideBarWrapper>
+      <SidebarCard title={t("State")}>Waiting</SidebarCard>
+      <SidebarCard title={t("Created By")}>User</SidebarCard>
+      <SidebarCard title={t("Created Time")}>
+        <>{currentRequest?.createdAt}</>
+      </SidebarCard>
```

Comment:
If the SidebarCard's content is ALWAYS text, just make a content prop like the title and pass in the value there.